webapp/tests/karma/ts/modules/tasks/tasks.component.spec.ts

```diff
@@ -16,6 +16,9 @@ import { TasksComponent } from '@mm-modules/tasks/tasks.component';
 import { NavigationComponent } from '@mm-components/navigation/navigation.component';
 import { Selectors } from '@mm-selectors/index';
 import { NavigationService } from '@mm-services/navigation.service';
+import {UserContactService} from '@mm-services/user-contact.service';
```

Comment:
```suggestion
import { UserContactService } from '@mm-services/user-contact.service';
```
Add these spaces pls